### DISCUSSION TITLE
Add Home Assistant entity_category support

### DISCUSSION THREAD

**Discussion by xannor** [tsightler/ring-mqtt#766] [Feature Requests]
It would be helpful if the [entity_category](https://www.home-assistant.io/integrations/sensor.mqtt/#entity_category) attribute could be added to the entity discovery message for entities that are configuration and/or diagnostic in nature. This would help reduce clutter when the HA entities is re-exported through other services (I.E. Home Bridge) as most will ignore categorized entities by default.

The following integrations support entity_category so it is not just limited to sensor:
[binary_sensor](https://www.home-assistant.io/integrations/binary_sensor.mqtt/#entity_category)
[switch](https://www.home-assistant.io/integrations/switch.mqtt/#entity_category)
[select](https://www.home-assistant.io/integrations/select.mqtt/#entity_category)
and many others...

**Comment by tsightler** [comment]
Feel free to submit a PR and I will review it.
